readme.md

```diff
@@ -54,30 +68,213 @@ Available environment variables:
 | HTTP_HOST | `0.0.0.0`     |                                                                                                           |
 | HTTP_PORT | `3000`        |                                                                                                           |
 
+### Routes
+
+| Method | Path                     | description                                                                         |
```

Comment:
```suggestion
| Method | Path                     | Description                                                                         |
```
and it would be nice to have the same case in all tables headers

Reply:
fixed